tasks/setup_static_dirs.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Create static dirs
+  ansible.builtin.file:
+    state: directory
+    mode: '0755'
+    owner: "{{ __galaxy_privsep_user_name }}"
+    group: "{{ __galaxy_privsep_user_group }}"
+    path: "{{ galaxy_themes_static_path }}/static-{{ subdomain.name }}"
+
+- name: Move all contents from static to parent directory
+  ansible.builtin.command: cp -r {{ galaxy_themes_static_dir }}/. {{ galaxy_themes_static_path }}/static-{{ \
```

Comment:
what's the reason we don't use a copy task here?

Reply:
I'm not sure now. I think I wanted to make sure it works recursively, but that also work with the `directory_mode` in the copy module. I can test it